app/src/containers/RegisterPage/index.js

```diff
@@ -723,7 +769,7 @@ class SignUpPage extends Component {
 
               <Button
                 //type="submit"
-                disabled={!this.state.termsAndConditionsAccept}
+                disabled={!isEnabled}
```

Comment:
consider using 
             disabled={!(this.state.termsAndConditionsAccept && this.state.confirmEmailChecked)}

but please consider using either `emailConfirmationError ` or `confirmEmailChecked` . I mean please get rid of either one of those since they are just boolean and basically the same

Reply:
I did think about this earlier but later I added function.